### PR TITLE
Add minvws-cibg pilot-2 credential to Yivi.

### PR DESCRIPTION
**Pull request by gerbrand-bosch**

### CODE REVIEW

**Comment by sanderhollaar** [comment]
can you maybe re-indent the file with two spaces for indentation, for readability?

File: minvws-cibg/description.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  <ID>minvws-cibg</ID>
+  <Name>
+	  <en>Authority for issuing UZI credentials</en>
+	  <nl>Authoriteit voor het uitgeven van UZI credentials</nl>
+  </Name>
+  <ShortName>
+	  <en>MINVWS CIBG</en>
+	  <nl>MINVWS CIBG</nl>
+  </ShortName>
+  <SchemeManager>pdbf</SchemeManager>
```
Comment (sanderhollaar) [inline minvws-cibg/description.xml:11]: s/pdbf/pbdf/

**Review by sanderhollaar — CHANGES_REQUESTED** [review]

File: minvws-cibg/Issues/pilot-2/description.xml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+	<SchemeManager>pdbf</SchemeManager>
+	<IssuerID>minvws-cibg</IssuerID>
+	<CredentialID>pilot-2</CredentialID>
+	<Description>
+		<en>Attributes of health identity from the UZI register</en>
+		<nl>Gegevens over uw zorgidentiteit uit het UZI-register</nl>
+	</Description>
+	<IssueURL>
+		<en>https://yivi.pilot.uzi-online.minvws.nl</en>
+		<nl>https://yivi.pilot.uzi-online.minvws.nl</nl>
```
[the remark this answers is not in the document]
Comment (gerbrand-bosch) [reply minvws-cibg/Issues/pilot-2/description.xml:22]: Yes, we're aware of this and planning to set this domain live tonight.

File: minvws-cibg/Issues/pilot-2/description.xml [hunk 3]
```diff
@@ -0,0 +1,117 @@
+<IssueSpecification version="4">
+    <RevocationServers>
+        <RevocationServer>https://yivi.pilot.uzi-online.minvws.nl/</RevocationServer>
```
[the remark this answers is not in the document]
Comment (gerbrand-bosch) [reply minvws-cibg/Issues/pilot-2/description.xml:3]: Yes, we're aware of this and planning to set this domain live tonight.

File: minvws-cibg/Issues/pilot-2/description.xml [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+	<Name>
+		<en>UZI register</en>
+		<nl>UZI-register</nl>
+	</Name>
+	<ShortName>
+		<en>UZI Register</en>
+		<nl>UZI-register</nl>
+	</ShortName>
+	<SchemeManager>pbdf</SchemeManager>
+	<IssuerID>MinVWS-cibg</IssuerID>
```
Comment (sanderhollaar) [inline minvws-cibg/Issues/pilot-2/description.xml:14]: s/MinVWS-cibg/minvws-cibg/

**Review by sanderhollaar — CHANGES_REQUESTED** [review]

File: minvws-cibg/Issues/pilot-2/description.xml (hunk 3 above)
Comment (sanderhollaar) [inline minvws-cibg/Issues/pilot-2/description.xml:3]: Revocation server of pbdf.minvws-cibg.pilot-2 should have no trailing /

**Review by sanderhollaar — CHANGES_REQUESTED** [review]

**Review by sanderhollaar — APPROVED** [review]

**Review by ivard — APPROVED** [review]

**Pull request merged**